CMakeLists.txt

```diff
@@ -1,60 +1,44 @@
-project(PROGRESSBAR_FOR_A_MORE_CIVILIZED_AGE)
-cmake_minimum_required(VERSION 2.8.5)
+project(elegant-progressbar-for-a-more-civilized-age-dev)
+cmake_minimum_required(VERSION 3.0.2)
 
 
 ###############################################################################
 # Using Clang is awesome
 # code based on http://stackoverflow.com/q/7031126
 ###############################################################################
-option(USE_CLANG "build application with clang" ON) # OFF is the default
+string(FIND ${CMAKE_CXX_COMPILER} "clang++" CXX_COMPILER_IS_CLANG++)
```

Comment:
since the compiler-flags are identical for g++ and clang, I would remove the "IF" completely, so this is not needed anymore